code/mob/living/critter/spider.dm

```diff
@@ -306,6 +343,21 @@
 			src.organHolder.drop_and_throw_organ("brain")
 			src.gib(1)
 
+	critter_attack(target)
+		if(ismob(target))
+			var/datum/targetable/critter/spider_bite/bite = src.abilityHolder.getAbility(/datum/targetable/critter/spider_bite)
+			var/datum/targetable/critter/clownspider_kick/kick = src.abilityHolder.getAbility(/datum/targetable/critter/clownspider_kick)
+			if (!kick.disabled && kick.cooldowncheck() && prob(20))
+				kick.handleCast(target)
+			else if(!bite.disabled && bite.cooldowncheck())
+				bite.handleCast(target)
+			else
+				src.set_a_intent(INTENT_HARM)
+				src.hand_attack(target)
```

Comment:
```suggestion
				..()
```

Reply:
can't do that here, parent is /spider not /critter, and so the parent call checks for abilities this doesn't have and does more behavior than wanted. If only there was a neat way to call a grandparent function.

Reply:
ahh right, I shouldn't do code review late at night. iirc there is a way to call grandparent functions by abusing .... but I'd argue that's worse than what's already there